README.md

```diff
@@ -255,7 +268,7 @@ come with Visual Studio Code by default.
 
 If you want to build legacy python code, please refer to [Building on legacy code and CUDA](https://github.com/pytorch/pytorch/blob/master/CONTRIBUTING.md#building-on-legacy-code-and-cuda)
 
-Build with CPU
+**Build with CPU**
```

Comment:
Can we somehow rename/rephrase it, say "Standard build/generic build, etc" As it is not not possible to build PyTorch without CPU (I would argue one could not even power computer on without CPU"

Reply:
Yeah, that whole section was a bit weird.  I believe it's talking about how to compile a non-cuda version of pytorch, which should have already been done in the OS specific sections right above this one.

The only difference between this section and the above ones is that it says to run `conda activate` first and then it has some details about OpenMP

Feels a bit redundant, and the CUDA section after this also needs a significant rewrite since it's written assuming a windows machine.

I don't understand the actual installation steps well enough to change the instructions just yet but in the long run I'd like to change the structure for the whole Install pytorch section to something like:

#### Install pytorch
There's two modes you can compile pytorch for, cpu-only computations and cuda-using computation mode.

If you want to use cuda, then first follow these instructions
To if you want to install a cpu-only version of pytorch, skip below to [this section](https://some.link)